nvme.c

```diff
@@ -7041,19 +7041,8 @@ static int dir_send(int argc, char **argv, struct command *cmd, struct plugin *p
 		}
 	}
 
-	struct nvme_directive_send_args args = {
-		.args_size	= sizeof(args),
-		.nsid		= cfg.namespace_id,
-		.dspec		= cfg.dspec,
-		.doper		= cfg.doper,
-		.dtype		= cfg.dtype,
-		.cdw12		= dw12,
-		.data_len	= cfg.data_len,
-		.data		= buf,
-		.timeout	= nvme_cfg.timeout,
-		.result		= &result,
-	};
-	err = nvme_directive_send(l, &args);
+	err = nvme_directive_send(l, cfg.namespace_id, cfg.doper, cfg.dtype, dw12, cfg.dspec, buf,
```

Comment:
Argument ordering is a bit off:
```suggestion
	err = nvme_directive_send(l, cfg.namespace_id, cfg.doper, cfg.dtype, cfg.dspec, dw12, buf,
```